contracts/src/v0.8/automation/dev/v2_2/AutomationRegistryBase2_2.sol

```diff
@@ -772,11 +760,6 @@ abstract contract AutomationRegistryBase2_2 is ConfirmedOwner {
       emit CancelledUpkeepReport(upkeepId, rawTrigger);
       return (false, dedupID);
     }
-    if (transmitInfo.upkeep.balance < transmitInfo.maxLinkPayment) {
-      // Can happen due to fluctuations in gas / link prices
-      emit InsufficientFundsUpkeepReport(upkeepId, rawTrigger);
```

Comment:
this will remove this event which the core node & plugin are listening to. we can remove from core node but it won't break anything if we don't

Reply:
yes good point, I was thinking of keeping the event definition so that nothing changes from plugin, o11y side, just that this event won't get emitted